client/pyre.py

```diff
@@ -37,6 +37,14 @@
 LOG: logging.Logger = logging.getLogger(__name__)
 
 
+def one() -> int:
```

Comment:
testing leftovers?

Reply:
Yeah, it's still not working. I'm trying to get results to show up in this PR.